src/Umbraco.Core/Constants-WebhookEvents.cs

```diff
@@ -81,6 +81,316 @@ public static class Aliases
             ///     Webhook event alias for media save.
             /// </summary>
             public const string MediaSave = "Umbraco.MediaSave";
+
+            /// <summary>
+            ///     Webhook event alias for document type changed.
+            /// </summary>
+            public const string DocumentTypeChanged = "documentTypeChanged";
```

Comment:
Not sure if it's a oversight it doesn't prefix these with `Umbraco.` like Content and Media events.

Reply:
@bjarnef I agree that it would have been nicer to use the `Umbraco.` prefix here too, but I used the existing aliases and just moved them to the constants class. I think changing the aliases will cause webhooks that are already configured to stop working after an upgrade and that's why I didn't change the aliases.